w03/d1/morning_exercise/main.js

```diff
@@ -1 +1,15 @@
 // Write a program that will `console.log` the numbers from 1 to 100. But for multiples of three `console.log` “Fizz” instead of the number and for the multiples of five `console.log` “Buzz”. For numbers which are multiples of both three and five `console.log` “FizzBuzz”.
+function fizzBuzz(num){
+for (var i =1; i<=100; i++){
+  if(i%5 ==0 && i%3 ==0){
+    console.log ("FizzBuzz");
+  }else if (i%5 ==0){
+    console.log ("Buzz");
+  }else if (i%3 ==0){
+    console.log ("fizz")
+  }else {
+    console.log(i);
+  }
+}
+}
+fizzBuzz(100);
```

Comment:
Hi!
Good job getting through the exercise. My only nitpick is to double check your indentation in your code. Proper indentation makes it easier to read your code. 
Thanks!